### PR TITLE
Lesson 4-4

### PR DESCRIPTION
**Pull request by mesomi**
Adding a CSS file and doing changes in the index.js file

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+body{
+  background-color: lavender;
+  color:indigo;
+  font-style: quicksand;
```
Comment (nicolealdurien) [inline css/index.css:4]: It looks like you were going for the extra step of bringing in a Google font here, but what shows on the page is still the default browser font.  Getting the font imported properly is just a couple more steps. On the [Google page](https://fonts.google.com/specimen/Quicksand?query=quicksand) for the font you want, just click the "Select this style" next to the version/font-weight you want:
 Screenshot from 2021-11-17 20-23-11

That will make this menu pop out from the side:
Screenshot from 2021-11-17 20-23-44

You'll need to copy that series of links under the `<link>` radio button, and paste it into the `<head>` area of your HTML (just like how you linked your CSS stylesheet). Then you'll need to copy and paste the CSS that starts with `font-family` into your CSS, in place of the `font-style` property you have here. [font-style](https://www.w3schools.com/cssref/pr_font_font-style.asp) generally isn't used for much beyond italicizing stuff.

File: css/index.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+#messages button {
+  background-color: plum;
+}
+footer {
+  background-color: yellowgreen;
+  letter-spacing: 2px;
+  color: whitesmoke;
+}
+
+@media screen and (max-width: 900px) and (min-width: 600px), (min-width: 1100px) {
```
Comment (nicolealdurien) [inline css/index.css:73]: This is actually very cool! :+1: :1st_place_medal:  At first I thought all of these size settings would conflict and break the media query, but when actually resizing my window I can see that you've effectively set 3 different breakpoints for 4 different styles:
Under 600 - browser default properties for font size and padding
601-900 - media query properties in use
901-1100 - browser defaults again
1101+ media query properties again

File: index.html [hunk 3]
```diff
@@ -4,20 +4,23 @@
...
   </head>
 
   <body>
     <h2>Melania Solis</h2>
       <section>
+        <h2>About</h2>
         <ul>
-          <li><h2>About</h2></li>
-          <p>I'm originally from <a href="https://en.wikipedia.org/wiki/Nicaragua">Nicaragua</a>.  I'm learning javascript with Code the Dream</p>
+          <li><p>I'm originally from <a href="https://en.wikipedia.org/wiki/Nicaragua">Nicaragua</a>.  I'm learning javascript with Code the Dream</p></li>
```
Comment (nicolealdurien) [inline index.html:15]: Having the `<p>` tags within your `<li>` doesn't hurt anything, but changes nothing about the appearance of your page versus having just the `<li>` tags.  Same thing in line 34.

**Review by nicolealdurien — COMMENTED** [review]
This is really coming together and looking good! :tada: I've added a few suggestions/extra info in comments at the spots they'd be relevant, but it's all super minor stuff/mostly doesn't impact the appearance of the page itself.

**Comment by mesomi** [comment]
Thank you so much for your feedback.  It really helps.

**Pull request merged**